taccsite_cms/static/site_cms/css/src/_imports/objects/o-section.css

```diff
@@ -237,3 +238,14 @@ Styleguide Objects.Section
 /* NOTE: A `z-index` > 0 is only necessary for sections before banner */
 /* CAVEAT: Any section pop-out el's can NOT overflow atop section after it */
 [class*="o-section--style"] { z-index: 1; }
+
+
+
+
+
+/* Scope (should be moved to `s-section`) */
+/* NOTE: FP-1318: Other should-be-scope CSS (<a>, <h1–6>) is in this file */
+
+[class*="o-section--style"] .c-callout__title {
+  color: var(--global-color-primary--xx-light);
+}
```

Comment:
This overwrites the color from the same file at https://github.com/TACC/Core-CMS/blob/fbb8c35/taccsite_cms/static/site_cms/css/src/_imports/objects/o-section.css#L100-L107 and https://github.com/TACC/Core-CMS/blob/fbb8c35/taccsite_cms/static/site_cms/css/src/_imports/objects/o-section.css#L122-L129

https://github.com/TACC/Core-CMS/blob/main/taccsite_cms/static/site_cms/css/src/_imports/objects/o-section.css